app/components/GettingStartedModal/GettingStartedModal.tsx

```diff
@@ -28,7 +28,7 @@ import type {
 const castedGettingStartedSchema = gettingStartedSchema as GettingStartedSchema;
 
 const GettingStartedModal = () => {
-    const [modalOpen, setModalOpen] = useState(() => !isGettingStartedModalDisabled());
+    const modalState = useModalState();
```

Comment:
Can we name it somewhat differently to suggest that it's the modal's _open_ state?

```suggestion
    const modalOpenState = useModalOpenState();
```

Reply:
fixed